pkg/bbgo/order_executor_general.go

```diff
@@ -300,17 +322,22 @@ func (e *GeneralOrderExecutor) OpenPosition(ctx context.Context, options OpenPos
 		submitOrder.Side = types.SideTypeSell
 		submitOrder.Quantity = quantity
 
-		Notify("Opening %s short position with quantity %f at price %f", e.position.Symbol, quantity.Float64(), price.Float64())
-		createdOrder, err2 := e.SubmitOrders(ctx, submitOrder)
-		if err2 != nil {
-			return err2
+		Notify("Opening %s short position with quantity %v at price %v", e.position.Symbol, quantity, price)
+		for {
```

Comment:
reduceQuantityAndSubmitOrders can be reused here

Reply:
done